Resources/doc/reference/troubleshooting.rst

```diff
@@ -94,6 +94,9 @@ Finally your settings in your ``sonata_media`` parameters will look like this:
                     - sonata.media.provider.file
 
                 formats:
+                    # You can leave this list empty. If you do the only format generated will be the `admin` format which is needed for the backend. This is useful if you use a third party service to create thumbnails and don't need to create static one using this bundle. 
```

Comment:
Please use `gqq` in vim on this line. Or use your editor and make sure to keep every line under 120 chars